intercity-server.rb

```diff
@@ -20,11 +24,41 @@ def usage
     puts "Commands:"
     puts "    help - Show the commands available"
     puts "    install - Run the setup for installing intercity-server"
+    puts "    restart - Restart your Intercity instance"
+    puts "    update - Update your Intercity instance"
   end
 
   def install
+    if installed?
+      puts "Intercity is already installed."
```

Comment:
@michiels Solved both points (Double bash install, and being aware of the CLI) with adding the text: "To update your Intercity instance, run: intercity-server update".